skimage/measure/fit.py

```diff
@@ -470,6 +470,19 @@ class EllipseModel(BaseModel):
         Ellipse model parameters in the following order `xc`, `yc`, `a`, `b`,
         `theta`.
 
+    Examples
+    -------
```

Comment:
Travis is unhappy about this docstring — I'm not sure if it's because there is a missing `-` at the end to make the dashes' width equal to that of "Examples".